### ISSUE TITLE
Text Box label not moving to the top when field is Autofilled

### ISSUE THREAD

**Issue by uvalleza** [SamProf/MatBlazor#590]
Hi all, I am having an issue that when your information is saved by your browser and you reload the page, the label seems to stay stuck. It isn't until you enter a letter that it moves properly to where it's supposed to be. See screenshots below. I am referring to the label. Any ideas on how to fix this? 

Incorrect when page loads:
![Incorrect]
(https://user-images.githubusercontent.com/64721355/83386904-abc30100-a3a0-11ea-8a3b-3ca503315832.PNG)

Correct when field is modified:
![Correct](https://user-images.githubusercontent.com/64721355/83387019-dc0a9f80-a3a0-11ea-9ad3-0a6af73e5de8.PNG)

**Comment by americanslon** [comment]
+1 
Been like that for as long as I know, it'd be nice to get this one sorted out.

**Comment by RonPeters** [comment]
I agree about the pre-filled issue, and this has been called out in some previous issues. We're working on that. However your screenshot seems to indicate something in addition to that. Are you still including Bootstrap from the default Blazor template? Bootstrap and Material are conflicting ways to solve similar problems. Be sure to remove Bootstrap from your project.

**Comment by uvalleza** [comment]
Hi Ron,

I have removed bootstrap but I have added to my site.css the below in order to prevent another issue i was having with autofill from browser changing the background color on the text field. Below fixed that issue but that pre-fill issue was always there even before using the below. Happens on every browser I've use.

 input:-webkit-autofill,
    input:-webkit-autofill:hover,
    input:-webkit-autofill:focus,
    input:-webkit-autofill:active {
        transition: background-color 5000s ease-in-out 0s;
    }

**Comment by americanslon** [comment]
@RonPeters any update on this?

**Comment by RonPeters** [comment]
@americanslon I've been absorbed in a product release and have not had a chance to address this issue. It is on my todo list for about 2 weeks from now.
